src/iGenVar.cpp

```diff
@@ -72,6 +72,16 @@ void initialize_argument_parser(seqan3::argument_parser & parser, cmd_arguments
                       "Specify the maximum allowed overlap between two alignment segments. "
                       "This value needs to be non-negative.",
                       seqan3::option_spec::advanced);
+    parser.add_option(args.sv_qual, 'q', "sv_qual",
+                      "Specify the minimum quality (amout of supporting reads) of a strucutral variant. "
```

Comment:
```suggestion
                      "Specify the minimum quality (amount of supporting reads) of a structural variant to be reported in the vcf output file."
```

The first two changes are typos that should be corrected also in `variant_output.hpp` and `iGenVar.hpp`. The last one is just a suggestion.